src/test/unit/mocks/birminghamAdministrativeCourtDailyCauseList.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
The schema are the same and these 4 are essentially the same list. We can use a single test file to avoid duplication

Reply:
using one file now.